ci_framework/roles/edpm_deploy/tasks/main.yml

```diff
@@ -68,6 +68,25 @@
       retries: "{{ cifmw_edpm_deploy_retries }}"
       delay: 20
 
+- name: Run gmake command manually
+  environment:
+    PATH: "{{ cifmw_path }}"
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+  ansible.builtin.shell: |
+      pushd /home/zuul/src/github.com/openstack-k8s-operators/install_yamls/devsetup
+      sed -i 's/-oc/id;oc/' Makefile
+      /usr/bin/gmake edpm_deploy_instance
+      popd
+  register: make_out
+
+- name: debug make output
```

Comment:
afaik this won't be hit in case of failure. Using the ci_make would allow to get proper logging even in case of failure.